x/tracing.go

```diff
@@ -0,0 +1,14 @@
+package x
+
+import (
+	"net/http"
+
+	"github.com/ory/x/otelx"
+)
+
+// TraceHandler wraps otelx.NewHandler, passing the URL path as the span name.
+func TraceHandler(h http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
@icyphox shouldn't otelx export this and we use it across all apps?

Reply:
Probably yeah. https://github.com/ory/x/pull/524

Reply:
Had to do the go.mod hack since ory/x v0.0.404 breaks pop:
```
# github.com/ory/x/popx
../../../go/pkg/mod/github.com/ory/x@v0.0.404/popx/migration_box.go:80:14: undefined: pop.NormalizeSynonyms
../../../go/pkg/mod/github.com/ory/x@v0.0.404/popx/migrator.go:307:29: m.Connection.NewTransactionContextOptions undefined (type *pop.Connection has no field or method NewTransactionContextOptions)
```